addons/payment_mercado_pago/controllers/main.py

```diff
@@ -0,0 +1,145 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import pprint
+
+from werkzeug.exceptions import Forbidden
+
+from odoo import _, http
+from odoo.exceptions import ValidationError
+from odoo.http import request
```

Comment:
```suggestion
from odoo.http import request, route
```
so that you can remove the global http import :)